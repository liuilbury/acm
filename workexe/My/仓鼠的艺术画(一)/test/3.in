55 73 71
9 1
40 28
32 58
38 59
8 9
20 38
30 39
35 5
5 28
25 48
51 73
48 29
20 22
53 53
7 57
51 32
22 68
45 17
22 26
51 58
27 71
25 17
22 12
53 9
5 40
9 10
41 69
13 5
49 27
49 71
55 19
20 6
37 3
55 44
36 2
42 28
19 16
15 52
23 30
10 44
17 58
32 57
1 22
39 22
49 60
24 7
53 21
51 61
49 57
55 43
20 8
30 39
46 24
3 45
22 28
24 58
12 34
15 26
13 3
44 29
14 60
51 57
21 56
14 42
52 11
50 64
29 16
47 45
28 65
38 32
19 51

