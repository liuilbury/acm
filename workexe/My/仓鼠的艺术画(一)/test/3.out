* * * * * * * * * * * * * * * * * * * * 43 43 43 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * 43 * * * * * * * * * * * * * * * * * * * * * * 54 * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 54 54 54 * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * 9 * * * * * * * * * * * 25 * * * * 54 * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * 9 9 9 * * * * * * * * * 25 25 25 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * 9 * * * * * * * * * * * 25 * * * * * * * * * * * * * * * * 15 * * * * * * * * * * * * * * * *
* * * * * * * * 5 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 15 15 15 * * * * * * * * * * * * * * *
1 * * * * * * 5 5 26 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 15 * * * * * * * * * * * * * * * *
1 1 * * * * * * 26 26 26 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 40 * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
1 * * * * * * * * 26 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 40 40 40 * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 57 * * * * * * * * * 40 * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * 59 * 28 * * * * * * * * * * * * * * * * * * * * * * * * * * * 57 57 57 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* 59 59 59 28 28 * * * * * * * * * * * * * * * * * * * * * * * * * * * 57 * * * * * * * 64 * * * * * * * * * * * * * * * * * 61 * * * * * * * * * * * * *
* * 59 * 28 * * * * * * * * * * * * * * * * * * * * 58 * * * * * * * * * * * * * * 64 64 64 * * * * * * * * 38 * * * * * * 61 61 61 * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * 58 58 58 * * * * * * * * * * * * * * 64 * * * * * * * * 38 38 38 * * * * * * 61 * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * 58 * * * * * * * * * * * * * * * * * * * * * * * * * 38 * * * * * 41 * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 41 41 41 * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * 37 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 71 * * * * * * 41 * * * * * * * * * * * * * * *
* * * * * 32 * 51 * * * * * * 37 37 37 * * * * 13 * * * * * * * * * * * * * * * 6 * * * * * * * * * * * 71 71 71 * * * * * * * * * * * * * * * * * * * * *
* * * * 32 32 51 51 51 * * * * * * 37 * * * * 13 13 13 * * * * * * * * * * * * * 6 6 6 * * * * * * * * * * * 71 * * * * 63 * * * * * * * * * * * * * * * * *
* * * * * 32 * 51 * * * 23 * * * * * * * * * 13 * * * 19 * 55 * * * * * * * * * 6 * * * * * * * * * * * * * * * * 63 63 63 * * * * * * * * * * 17 * * * * *
* * * * * * * * * * 23 23 23 * * * * * * * * * * * 19 19 55 55 55 39 * * * * * * * * * * * * * * * * * * * * * * * * * 63 * * * * * * * * * * 17 17 17 * * * *
* * * * * * 46 * * * * 23 * * * * * * * * * * * * * 19 * 55 39 39 39 * * * * * * * * * * * * * * * * * * * * * * * * * * 56 * * * * * * * * * 17 * * * * *
* * * * * 46 46 46 * * * * * * * * 22 * * * * * * * * * * * * 39 * * * * * * * * * * * * * * * * * 10 * * * * * * * * 56 56 56 * * * * * * * * * * * * * *
* * * * * * 46 * * * * * * * * 22 22 22 * * * * * * * * * * * * * * * * * * * * * * * * * * * * 10 10 10 * * * * * * * * 56 * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * 22 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 10 * * * * * * * * * * * * * * * * * * * * * * 21 * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 69 * * * * 21 21 21 *
* * * * * * * * * * * * * * * 67 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 69 69 69 * * * * 21 * *
* * * * * * * * * * * * * * 67 67 67 * * * * * * * * * * * * * * * * * * * * * 52 * * * * * * * * * * * * * * * * * * * * * * * * * 69 * * * * * * * *
* * * * * * * * * * * * * * * 67 * * * * * * * * * * * * * * * * * * * * * 52 52 52 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 52 * * * * * * * * * * * * * * * * * 42 3 * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 42 42 42 3 * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 42 3 * * * * * * * * * * * * * * *
* * * * 8 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* 35 * 8 8 8 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
35 35 35 * 8 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* 35 33 33 * * * * * * * * * * * * * * * * * * * * * * * * * * * 70 * * * * * * * * * * * * * * * * * * * * * * * * * * 4 * * * * * * * * * * * * * *
* * 33 * * * * * * * * * * * * * * * * * * 44 * * * * * * * * 70 70 70 * * * * * * * * * * * * * * * * * * * * * * * * 4 4 4 * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * 44 44 44 * * * * 2 * * * 70 * * * * * * * * * * * * * * * * * * * * * * * * * * 4 * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * 44 * * * * 2 2 2 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 27 * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * 36 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 27 27 27 * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * 36 36 36 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 27 * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * * 36 60 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * 18 * * * * * * * * * * 60 60 60 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * 18 18 18 * * * * * 53 * * * * 60 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * 18 * * * * * 53 53 53 * * * * * * * * * * * * * * * * * * * 68 * * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * 53 * * * * 12 * * * * * * * * * * * * * * 68 68 68 * * * * * * * * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * * * * * * * * * * 29 12 12 12 * * * * * * * * * * * * * * 68 * * * * * * * * * * * 49 * * 45 * * * * * * * * * * 30 * *
* * * * * * * * * * * * * * * * * * * * * * * * * 29 29 29 12 * * * * * * * * * * * * * * * * * * * * * * * * * * 49 49 49 45 45 45 * * 66 * * * * * 30 30 30 *
* * * * * * * * * * * * * * * * * * * * * * * * * * 29 * * * * 16 * * * * * * * * * * * * * * * * * * * * * * * * 62 20 * 45 48 * 66 66 66 * * * * * 30 * 11
* * * * * * * * * * 65 * * * * * * * * * * * * * * * * * * * 16 16 16 * * * * * * * * * * * * * * * * * * * * * * 62 62 62 20 48 48 48 * 66 * * * * * * * 11 11
* * * * * * * * 24 65 65 65 * * * * * * * * 47 * * * * * * * * * * 16 * * * * * * * * * * * * * * * * * * * * 14 * * * 62 20 * * 48 * * * * * * * * * * * 11
* * * * * * * 24 24 24 65 * * * * * * * * 47 47 47 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * 14 14 14 * * * * * * * * * * * * * * * * * * *
* * * * * * * * 24 * * * * * * * * * 31 * 47 * * * * * * * * * * * * * * * * * * * * * 50 34 * * * * * * * * 14 * * * * * * * * * * * * * * * * * * * *
* * * * * * * * * * * * * * * * * 31 31 31 * * * * * * * * * * * * * * * * * * * * * 50 50 50 34 * * * * * * * * * * * * * * * * * * * * * * * * * * * *
